website/client-old/js/services/questServices.js

```diff
@@ -104,6 +104,9 @@ angular.module('habitrpg')
             Groups.data.party = party;
             $state.go('options.social.party');
             resolve();
+            if ($state.current.name === "options.social.party") {
```

Comment:
Is there a reason this is comes after the promise is resolved? I have this PR on my list to try.